src/app/base/components/GlobalSideNav/_index.scss

```diff
@@ -0,0 +1,166 @@
+@mixin GlobalSideNav {
+  .l-navigation {
```

Comment:
I'd suggest you create another wrapper inside of the main navigation and try not to modify `.l-navigation` itself. In general we should avoid overwriting vanilla styles as much as possible.

Reply:
I can do that yeah.